examples/simple_solution.xml

```diff
@@ -0,0 +1,99 @@
+<database
+        xmlns="http://calphad.org/thermml/0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:xi="http://www.w3.org/2001/XInclude"
+        xsi:schemaLocation="http://calphad.org/thermml/0.0 ../schema/thermml-schema.xsd"
+
+        name="Example Simple Solution Database">
+
+    <description>
+        This is an example thermodynamic database for demonstrating the ThermML schema for simple solutions.
+    </description>
+
+    <nonChemicalPotentials>
+        <nonChemicalPotential id="potT" symbol="T" units="K"/>
+        <nonChemicalPotential id="potP" symbol="P" units="bar"/>
+        <nonChemicalPotential id="potB" symbol="B" units="T"/>
+    </nonChemicalPotentials>
+
+    <systemComponents>
+        <systemComponent symbol="Va" />
+        <systemComponent symbol="C"  refstate="GRAPHITE" molar_mass="12.011"  h298="1054.0"   s298="5.7423"  />
+        <systemComponent symbol="Fe" refstate="BCC_A2"   molar_mass="55.847"  h298="4489.0"   s298="27.2797" />
+        <systemComponent symbol="Mn" refstate="CBCC_A12" molar_mass="54.9380" h298="4995.696" s298="32.2206" />
+    </systemComponents>
+
+    <phases>
+        <phase name="FCC_A1" alias="Austenite" state="solid" xsi:type="SimpleSolutionPhaseType">
+            <structure spacegroup="225" prototype="NaCl">
```

Comment:
Is `<structure>` only for solid phases?

Reply:
I feel like this is a tougher question than I thought at first. The thing is, very different phase types do map to the same thermodynamic 'concepts'. I feel that a 'strict' categorization via the `xsi:type` would be very hard to enforce and do, since a lot of the abstractions that are being used are rather vendor-specific.

Therefore, I simply assume that the `<structure>` node would be optional? And that there would be more optional nodes for other "subtypes" of phases?

Since none of the GEMs actually care for the state of a phase, I am not even sure if we want to e.g. enumerate the possible states, or have this an obligatory field. E.g. in SGTE databases, phases are noted as being `condensed` and `solid`. These are experimental classifications, of which I imagine others may be around, too. 

What do you think?

Reply:
OK